src/game.py

```diff
@@ -38,7 +44,9 @@ def gamelogic(self, guessed_letter):
         lives(int),display(str),badletter(str)
         """
         letters = list(self.word)
-        dis = ['_ ' for i in range(len(self.word))]
+        dis = ['_' for i in range(len(self.word))]

```

Comment:
Replace with `dis = ["_"] * len(self.word)`

I would suggest saving `len(self.word)` in `self.wlen`